client/layout/masterbar/notifications.jsx

```diff
@@ -23,32 +30,51 @@ export default React.createClass( {
 		tooltip: React.PropTypes.string,
 	},
 
+	componentDidMount() {
+		wpcom
```

Comment:
the plan here is to pull wpcom out of the notes client entirely so that it's simply a view over a list of notification data items.

somewhere outside of the `<Notifications { ...{ notifications, actions } } />` component would be the poller/Pinghub client updating the notes inside of Redux app state and the actions would be a mapping between action type and Redux action dispatched

Example
```js
{
	likePost: ( siteId, postId ) => dispatch( likePostAction( siteId, postId ) ),
}